packages/ia-components/components/audio-players/archive-audio-jwplayer-wrapper.jsx

```diff
@@ -92,13 +92,16 @@ class ArchiveAudioPlayer extends Component {
       return;
     }
     const incomingTrackChange = index !== prevIndex;
+    const playersCurrTrack = jwplayerInstance.getPlaylistIndex();
+    const playerIncomingTrack = index - 1;
+    const nextTrackPlaying = playersCurrTrack === playerIncomingTrack;
```

Comment:
TEENSY TINY SUGGESTION: I'm not quite positive from reading the names what this state represents. Does `nextTrackPlaying` mean that the next queued up track is currently playing? Or is this the next track _to be played_? Or something else?

Also, is `playerIncomingTrack` the track that is... umm... specified in the URL? But not playing yet? i.e. Where is it incoming from?

These tiny little clarifications will help a reader understand your intent. :)

Reply:
thanks for the push for clarity,

`nextTrackPlaying` is specific to auto advancing that happens so i'll update as `autoAdvanceHappening`.

`playerIncomingTrack` converts the react's track index to jwplayer's so i'll update as `incomingTrackAsJwpTrack`